### ISSUE TITLE
The board is not right for the black player

### ISSUE THREAD

**Issue by UmairJibran** [code100x/chess#138]
a1 should be black for white and for black alike.

right now it's correct for white, but not so for black.

<img width="1678" alt="Screenshot 2024-04-21 at 01 53 21" src="https://github.com/code100x/chess/assets/43789374/c2430ad9-c3b5-4b7a-ac22-43c350b9eca2">

**Comment by nimit9** [comment]
No it's h8 for black always. Please take a look at chess.com, the notation remain same on the board regardless of the orientation

**Comment by nimit9** [comment]
No it's h8 for black always. Please take a look at chess.com, the notation remain same on the board regardless of the orientation

**Comment by nimit9** [comment]
Oh, my bad,

The notations are correct as they're supposed to be on the chess board.

The colors on the piece have been alternated. Can someone fix this please, just the colors.

**Comment by UmairJibran** [comment]
yep, they are confusing when looking at them from both sides, I wrote and deleted the issue twice before finally creating it 😅

**Comment by nimit9** [comment]
> yep, they are confusing when looking at them from both sides, I wrote and deleted the issue twice before finally creating it 😅

Yep the colors are alternated.
